sympy/matrices/matrices.py

```diff
@@ -3084,6 +3084,44 @@ def eigenvects(self, **flags):
                 out.append((r, k, [mat._new(b) for b in basis]))
         return out
 
+    def left_eigenvects(self, **flags):
+        """Returns left eigenvectors and eigenvalues.
+
+        This function returns the list of triples (eigenval, multiplicity,
+        basis) for the left eigenvectors. Options are the same as for
+        eigenvects(), i.e. the **flags arguments gets passed directly to
```

Comment:
Sphinx does something weird with `**flags`, so you should put it in double backquotes.